src/sentry/migrations/0186_add_externalactor.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 1.11.29 on 2021-04-13 16:46
+
+import django.db.models.deletion
+import django.utils.timezone
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+import sentry.types.integrations
+
+
+class Migration(migrations.Migration):
+
```

Comment:
Looks like our usual boilerplate with atomic/is_dangerous etc is missing, did you remove it?

Reply:
hmm that's weird I didn't remove it...at least not intentionally I can re-generate

Reply:
update for the masses: it was https://github.com/getsentry/sentry/pull/25236